variables.tf

```diff
@@ -171,3 +177,9 @@ variable "tags" {
   default     = {}
   description = "A mapping of tags to assign to the logs bucket. Please note that tags with a conflicting key will not override the original tag."
 }
+
+variable "versioning_enable" {
```

Comment:
Naming for conditionally enabling things is `enable_<feature>`. 
* https://github.com/trussworks/terraform-aws-s3-private-bucket/blob/master/variables.tf#L66
* https://github.com/trussworks/terraform-aws-config/blob/master/variables.tf#L288
* https://github.com/trussworks/terraform-aws-ses-domain/blob/master/variables.tf#L17